tests/foreman/ui/test_registration.py

```diff
@@ -328,7 +335,7 @@ def test_global_registration_form_populate(
         cmd = session.host.get_register_command(
             {
                 'general.organization': new_org.name,
-                'general.operating_system': default_os.title,
+                'general.activation_keys': new_ak.name,
```

Comment:
We can skip AK selection here?

Reply:
We cannot skip it.
The issue with it is airgun doesn't refresh the page when we are navigating to the same page again. If you see the previous command, we are testing the register command and selecting an org doesn't refresh the page. So, the activation key doesn't get auto populated. Hence we need to select it explicitly.

Reply:
selecting the org is different it doesn't refresh I know, but for AK when we navigate to Register page for the first time, AK should be auto-populated if its single AK in org, as per recent feature, which here should work as `new_org` contains only single `new_ak` AK 
 I'm not sure how you're not able to reproduce it, or you might've some more AK due to improper teardown of some test

Reply:
It's cannot be possible to have another AK in the new org as we are creating in the test itself. Also, we don't delete org and loc explicitly in the teardown.